src/components/CardChoice/CardChoice.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { PropsWithChildren } from "react";
+import { Icon } from "components/Icon/Icon";
+
+type IntrinsicElements = JSX.IntrinsicElements["a"];
+
+interface Props extends IntrinsicElements {
+  actionText?: string;
+  bodyText?: string;
+  bordered?: boolean;
+  className?: string;
+  darkBG?: boolean;
+  headingText?: string;
+}
+
+/**
+ * CardChoice Component
+ *
+ * CardChoice is wrapped by an anchor tag. All unspecified props will be provided to the anchor.
+ * @param {string}             actionText     Optional text prompt to be displayed left of the navigation arrow.
+ * @param {boolean}            bordered       Renders a border around the CardChoice. Applying borderd on CardChoice will override the CardChoiceGroup.
+ * @param {string}             bodyText       Text to be rendered in the body.
+ * @param {React.ReactNode}    children       Children provided will be rendered below the bodyText.
+ * @param {string}             className      Additional classes that can be applied to the root element.
+ * @param {boolean}            darkBG         Renders the CardChoice with a darker background. Applying darkBG on CardChoice will override the CardChoiceGroup. CardChoiceGroup has an option for alternatingBG.
+ * @param {string}             headingText    Bolded heading text displayed at the top of the CardChoice.
+ * @param {string}             href           href provided to the anchor.
+ * @param                      onClick        onClick provided to the anchor.
+ */
+export const CardChoice: React.FC<PropsWithChildren<Props>> = ({
+  actionText,
+  bodyText,
+  bordered,
+  children,
+  className,
+  darkBG,
+  headingText,
+  href,
+  onClick,
+  ...rest
+}) => {
+  const classes = `card-choice${darkBG ? " card-choice--dark" : ""}${
+    bordered ? " card-choice--bordered" : ""
+  }${className ? ` ${className}` : ""}`;
+  return (
+    <a {...rest} className={classes} href={href} onClick={onClick}>
+      <div className="content">
+        <p className="heading">{headingText}</p>
```

Comment:
I'm thinking this could be a "H" tag for the semantics of a heading? Or at least the heading role:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role

Reply:
I'm just gonna set the role for now. If you have a particular level of heading you think it should be I can make that change, I just didn't want to do something stupid.

Reply:
Reading the docs now and it's saying that this is effectively the same as just using a heading. I'll just set it h6 and if we need to change it we can.